neo/io/proxyobjects.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, array_annotations=None, **annotations):
         if array_annotations is None:
             array_annotations = {}
         for k, v in array_annotations.items():
-            array_annotations[k] = np.asarray(k)
+            array_annotations[k] = np.asarray(v)
```

Comment:
I really did this ?

Reply:
Nope, that's my typo, but somehow I was not allowed any more to push to your branch ;)